tools/scenario-geojson2scn.py

```diff
@@ -0,0 +1,240 @@
+    elif "name" in feature["properties"]:
+        id = normalize_id(feature["properties"]["name"])
+    else:
+        id = f"Area{area_id}"
+        area_id = area_id + 1
+    return id
+
+
+def write_cmd(cmd: str, *arguments: List[str], timestamp: timedelta = timedelta(seconds=0)):
+    hours, remainder = divmod(timestamp.seconds, 3600)
+    minutes, seconds = divmod(remainder, 60)
+    timestamp = "{:02}:{:02}:{:02}".format(int(hours), int(minutes), int(seconds))
+    return f'{timestamp}> {cmd.upper()} {" ".join(str(arg).upper() for arg in arguments)}'
+
+
+def write_aircrafts(config, features):
+    def get_point_id(feature, idx):
+        return f"{get_id(feature)}_{idx}"
+
+    def get_line_points(feature):
+        points = list(p for p in feature["geometry"]["coordinates"])
+        for idx, point in enumerate(points):
+            if len(point) > 2:
+                points[idx] = [*reversed(point[0:2]), point[2]]
+            else:
+                points[idx] = [*reversed(point[0:2]), None]
+        return points
+
+    config["ac_files"] = []
+    for feature in features:
+        props = feature["properties"]
+
+        if feature["geometry"]["type"] == "LineString" and "model" in props:
+            aid = get_id(feature)
+            pid = get_point_id(feature, 0)
+            if "start" in props:
+                start = normalize_id(props["start"])
+            else:
+                start = pid
+
+            out = []
+
+            points = get_line_points(feature)
+            point_idxs = []
+            for idx, point in enumerate(points):
+                point_idxs.append(idx)
+                out.append(write_cmd("defwpt", get_point_id(feature, idx), *point[:2], "fix"))
+
+            out.append(write_cmd("cre", aid, props["model"], start, 0, 0, 0))
+
+            if "repeat" in props:
+                if props["repeat"] == "loop":
+                    point_idxs = point_idxs * 5
+                elif props["repeat"] == "return":
+                    point_idxs = [*point_idxs, *reversed(point_idxs[:-1])]
+
+            d = timedelta(seconds=1)
+            for idx in point_idxs:
+                point = points[idx]
+                cmd = write_cmd("addwpt", aid, get_point_id(feature, idx), timestamp=d)
+                if point[2] is not None:
+                    cmd = f"{cmd}, {point[2]}"
+                out.append(cmd)
+
+            # fmt: off
+            d = timedelta(seconds=2)
+            out.append(write_cmd("alt", aid, props["alt"], timestamp=d))
+            out.append(write_cmd("vs", aid, 800, timestamp=d))
+            out.append(write_cmd(aid, "atalt", props["alt"], "spd", aid, props["vs"], timestamp=d))
+            out.append(write_cmd("vnav", aid, "on", timestamp=d))
+            out.append(write_cmd(aid, "at", pid, "do", aid, "atdist", start, "0.03", "spd", aid, 10, timestamp=d))
+            out.append(write_cmd(aid, "at", pid, "do", aid, "atdist", start, "0.001", "spd", aid, 0, timestamp=d))
```

Comment:
The distance here should be in relation to the endpoint, not the starting point.
These might of course be one and the same, depending on the operation.
```suggestion
            out.append(write_cmd(aid, "at", pid, "do", aid, "atdist", end, "0.03", "spd", aid, 10, timestamp=d))
            out.append(write_cmd(aid, "at", pid, "do", aid, "atdist", end, "0.001", "spd", aid, 0, timestamp=d))
```